dynawo/sources/API/CRT/CRTCriteria.h

```diff
@@ -170,6 +176,19 @@ class Criteria {
    * @return a component_id_const_iterator to the end of components ids container
    */
   virtual component_id_const_iterator end() const = 0;
+
+  /**
+   * @brief Test if this criteria is limited to a specific country
+   * @return @b true if this criteria has at least one country filter
+   */
```

Comment:
criteria -> criterion.
And adapt the brief part.

Reply:
Done